### DISCUSSION TITLE
Streaming with face filter

### DISCUSSION THREAD

**Discussion by yogeshJain0727** [ant-media/Ant-Media-Server#6207] [General]
We want to implement a face filter so that streamers can go live using this filter. On broadcasting, users can change the face filter. Please suggest how to implement this. We don't want to use deepar.ai or any paid service, and we want this implemented on both web and app platforms.

**Comment by Mohit-3196** [comment]
Hi @yogeshJain0727,
How are you?

Thank you for reaching out with your question. While implementing a face filter for streamers to use on both web and app platforms sounds like an exciting feature, it's important to note that this involves client-side implementation rather than being directly related to the Ant Media Server itself.

I would recommend exploring libraries or frameworks that specialize in computer vision and facial recognition, as these are crucial components for developing face filters. Open-source options like OpenCV or TensorFlow can be valuable resources for this purpose. Additionally, leveraging WebGL for web platforms and native libraries for app development can provide the necessary tools for real-time processing and rendering.

Should you require further assistance or guidance during the implementation process, both the community and the Ant Media Server developers are here to support you. Feel free to delve deeper into your research and experimentation, and don't hesitate to ask for help when needed.

Thank you for your inquiry, and best of luck with your project!
